languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -9007,6 +9005,31 @@ though, and has since been slightly modified)-->
         <example correction="trois fois par semaine">Je joue au Softball <marker>trois fois au semaine</marker>.</example>
       </rule>
       <!-- End of additions -->
+        <rulegroup id="YA" name="ya" default="temp_off">
+            <rule>
+                <pattern>
+                    <token>il</token>
+                        <marker>
+                            <token>ya</token>
+                        </marker>
+                </pattern>
+                <message>Ces deux mots doivent être séparés : <suggestion>y a</suggestion>.</message>
+                <example correction="y a">Il <marker>ya</marker> douze personnes.</example>
+            </rule>
+            <rule>
+                <antipattern>
+                    <token>il</token>
+                    <token>ya</token>
+                </antipattern>
+                <pattern>
+                        <marker>
+                            <token>ya</token>
+                        </marker>
+                </pattern>
+                <message>Cette formulation est considérée familière. Vouliez-vous dire <suggestion>y a</suggestion> ?</message>
+                <example correction="Y a"><marker>Ya</marker> douze personnes.</example>
+            </rule>
+        </rulegroup>
       <rule id="COMME_ME_LE_DIRE" name="comme me le dire accord">
```

Comment:
I don't see the point of the two rules for "ya". The suggestion is the same in both rules. Is "ya" sometimes colloquial (as it is implied in the second rule) or is it always wrong? 

And what about "y'a"? We need something similar to avoid this duplicate suggestion: 
![imatge](https://user-images.githubusercontent.com/4289501/92649391-3e9e0080-f2eb-11ea-9fb7-c1758e794cf3.png)

Reply:
_ya_ and _y'a_ are both written colloquialisms. They can occur with and without _il_ before them. @vkyfox, I think we need to differentiate: when there's an _il_ before _ya_ or _y'a_, we can treat it like a typo, no need to acknowledge the colloquial form. When there's no _il_ before _ya_ or _y'a_, your message is perfect, of course.

Reply:
I updated the pull request, it should be good now.